sympy/geometry/polygon.py

```diff
@@ -402,6 +403,59 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
+    def second_moment_of_area(self, p=None):
+        """Second and product moment of area of polygon.
+
+        Parameters
+        ==========
+
+        p : Point, None
```

Comment:
Shouldn't this be `p : Point2D, None`?

Reply:
a, b  = symbols('a, b')
p = Point2D(a, b)
p =  Point(a, b)
p = (a, b)
it works in every case

Reply:
![screenshot from 2017-12-13 10-53-51](https://user-images.githubusercontent.com/23614977/33922902-f097b1fe-dff3-11e7-8b31-c7a3e3774d07.png)

Reply:
Ok great. Please make this explicit then. Maybe:

```
p : Point, two-tuple of sympifiable objects, or None
```